k8s/aat/common/ccd/api-gateway-web.yaml

```diff
@@ -23,6 +23,7 @@ spec:
       image: hmctspublic.azurecr.io/ccd/api-gateway-web:prod-be19da52
       environment:
         CORS_ORIGIN_WHITELIST: "*"
+        TIMING-ALLOW-ORIGIN: "https://www-ccd.aat.platform.hmcts.net"
```

Comment:
Please remove protocol.
Correct value should be: `www-ccd.aat.platform.hmcts.net`
Please apply the same to all.

Reply:
Sorry, please ignore the above.